src/vs/workbench/contrib/typeHierarchy/common/typeHierarchy.ts

```diff
@@ -168,7 +168,13 @@ CommandsRegistry.registerCommand('_executePrepareTypeHierarchy', async (accessor
 });
 
 function isTypeHierarchyItemDto(obj: any): obj is TypeHierarchyItem {
-	return true;
+	const item = obj as TypeHierarchyItem;
+	return typeof obj === 'object'
+		&& typeof item.name === 'string'
+		&& !!item.kind
```

Comment:
In TS how do we check if it's a valid enum?  
I think `!!item.kind` is not working because value of `SymbolKind` can be `0`, and `!!0` is false.

Reply:
`typeof fff === 'number'` should be enough

Reply:
got it.